db/migrate/20170317155628_add_details_to_splits.rb

```diff
@@ -0,0 +1,8 @@
+class AddDetailsToSplits < ActiveRecord::Migration
+  def change
+    add_column :splits, :hypothesis, :text, null: true
+    add_column :splits, :assignment_criteria, :text, null: true
+    add_column :splits, :description, :text, null: true
+    add_column :splits, :squad_owner, :string, null: true
```

Comment:
It seems to me that, broadly, engineering teams would be capable of identifying the owner of a given split test, so the idea of a split having an owner is perfectly reasonable. I think I would just drop the "Squad" part from the name since it's not really something that means anything to the TestTrack product. And if owner is ambiguous in the UI, then we should feel comfortable adding another example text hint that clarifies the kinds of values that would be reasonable for this free text field.